demos/dialog.html

```diff
@@ -166,17 +132,17 @@ <h2 id="mdc-dialog-with-list-label" class="mdc-dialog__header__title">
               </ul>
             </section>
             <footer class="mdc-dialog__footer">
-              <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--cancel">Decline</button>
-              <button type="button" class="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--accept">Accept</button>
+              <button type="button" class="mdc-button demo-button mdc-dialog__footer__button mdc-dialog__footer__button--cancel">Decline</button>
+              <button type="button" class="mdc-button demo-button mdc-dialog__footer__button mdc-dialog__footer__button--accept">Accept</button>
             </footer>
           </div>
           <div class="mdc-dialog__backdrop"></div>
         </aside>
       </div>
       <section class="example">
-        <button id="default-dialog-activation" class="mdc-button mdc-button--raised">Show Dialog</button>
-        <button id="colored-footer-button-dialog-activation" class="mdc-button mdc-button--raised">Show Colored Footer Button Dialog</button>
-        <button id="dialog-with-list-activation" class="mdc-button mdc-button--raised">Show Scrolling Dialog</button>
+        <button id="default-dialog-activation" class="mdc-button demo-button mdc-button--raised demo-button--raised">Show Dialog</button>
```

Comment:
not sure how you feel about these classes, but needed them because of some class selectors in common.scss

Reply:
Yeah, that's fine. I would just order them so `demo-` is next to `demo-`

It's only a matter of style, but I (and Ken) feel this is more readable.